hw1_classes/code/task4.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+matrix = []
+for idx in range(0, 100, 10):
```

Comment:
Не увидел ни отдного `__call__`, например
```suggestion
for idx in range.__call__(0, 100, 10):
```